src/freenet/clients/http/FProxyToadlet.java

```diff
@@ -139,7 +140,7 @@ public void handleMethodPOST(URI uri, HTTPRequest req, ToadletContext ctx) throw
 		}
 	}
 
-	public static void handleDownload(ToadletContext context, Bucket data, BucketFactory bucketFactory, String mimeType, String requestedMimeType, String forceString, boolean forceDownload, String basePath, FreenetURI key, String extras, String referrer, boolean downloadLink, ToadletContext ctx, NodeClientCore core, boolean dontFreeData, String maybeCharset) throws ToadletContextClosedException, IOException {
+	private void handleDownload(ToadletContext context, Bucket data, BucketFactory bucketFactory, String mimeType, String requestedMimeType, String forceString, boolean forceDownload, String basePath, FreenetURI key, String extras, String referrer, boolean downloadLink, ToadletContext ctx, NodeClientCore core, boolean dontFreeData, String maybeCharset) throws ToadletContextClosedException, IOException {
```

Comment:
A `grep` over my source code directory (which contains quite a number of our plugins) does not reveal any hits for `handleDownload`. However, yes, with the way we currently expose *everything* to plugins changing *anything* might break a plugin.